Packs/WolkenITSM/Integrations/WolkenITSM/WolkenITSM.yml

```diff
@@ -0,0 +1,310 @@
+category: Case Management
+commonfields:
+  id: Wolken ITSM
+  version: -1
+configuration:
+- defaultvalue: api-brdcmitsmbst.wolkenservicedesk.com
+  display: Server URL (e.g. api-brdcmitsmbst.wolkenservicedesk.com)
+  name: url
+  required: true
+  type: 0
+- display: API Key
```

Comment:
Please use credentials param, see https://xsoar.pan.dev/docs/integrations/code-conventions#credentials

Reply:
I have changed parameter named as API key to credentials. Thank You..!!